orientdb/src/main/scala/akka/stream/alpakka/orientdb/OrientDBSinkSettings.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.orientdb
+
+//#sink-settings
+final case class OrientDBSinkSettings(maxPartitionSize: Int = Runtime.getRuntime.availableProcessors(),
+                                      maxPoolSize: Int = -1,
+                                      maxRetry: Int = 1,
+                                      retryInterval: Int = 5000,
+                                      bufferSize: Int = 10,
+                                      dbUrl: String,
```

Comment:
Declaring non-optional parameters after optional parameters doesn't sound good to me.